book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,180 @@
+# Simple Payment and State Verification
+
+The transaction contains a signature. Validators store those signatures for a
+short period of time to ensure the transaction is not processed more than once.
+Validators provide a JSON RPC endpoint, which clients can use to query the
+cluster if a transaction has been recently processed. Validators also provide a
+PubSub notification, whereby a client registers to be notified when a given
+signature is observed by the validator. While these two mechanisms allow a
+client to verify a payment, they are not a proof and rely on a trusted
+validator.
+
+## Light Clients
+
+Light clients are cluster participants that are not running a fullnode.  The
+goal of this proposal is to provide a level of security that is greater than
```

Comment:
Hmm, that's not the goal of this proposal, that's the goal of the implementation, which this proposal describes.
How about rewrite this paragraph as:
"This proposal describes a 'light client', a cluster participant that does not itself run a fullnode. This light client would provide a level of security greater than trusting a remote fullnode, without requiring the light client to spend a lot of resources verifying the ledger"

Reply:
@t-nelson ^ want to take over landing this :)

Reply:
Ha!  Sure.

I'll steal your branch and make a new PR then?

Reply:
@t-nelson you can push directly to this branch, I think...
![image](https://user-images.githubusercontent.com/40183961/60123271-e9082300-973b-11e9-8a8d-13a5d5a292c6.png)

Reply:
Permission to do more editing, @t-nelson ? 👼

Reply:
Err... This one.  Yup!  Go for it!

Reply:
Okay, all from me for now :)